samples/csharp/getting-started/MulticlassClassification_HeartDisease/HeartDiseaseConsoleApp/DataStructure/HeartData.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace MulticlassClassification_HeartDisease.DataStructure
+{
+    public class HeartData
+    {
+        public float Age;
+        public float Sex;
+        public float Cp;
```

Comment:
Out of cusiosity, could you explain the meaning of each of these fields? Other than "Age", "Sex" I have no clue what those columns are. It would be great to test the model with my own data if I ever get it from a blood analysis, I guess?..  ;)

Or point to any place where the dataset is explained?